troubleshooting/troubleshooting-ios-cannot-load-iframe.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Cannot Load iframe on iOS devices when using Cordova 5.0.0 or later
+description: Cannot load iframe on iOS devices with Cordova 5.0.0 and 5.2.0.
+type: troubleshooting
+page_title: Cordova 5.0.0 or later apps do not load iframe on iOS
+slug: iframe-not-loading-ios
+position: 
+tags: appbuilder, ios, cordova 
+teampulseid: 
+ticketid: 1060248
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+  <tr>
+    <td>Cordova framework version</td>
+    <td>Cordova 5.0.0 and above</td>	
+  </tr>
+</table>
+
+## Description
+
+Apps that are deployed on an iOS device don't show iframe's content.
+
+## Steps to Reproduce
+
+1. Create a Cordova application.
+1. Set the **Cordova version** to be equal or greater than **5.0.0** from **Properties->Cordova** Version.
+1. Include a `<iframe src=""></iframe>` tag to a HTML view.
+1. Deploy the app to an iOS device.
+
+## Cause
+
+With the introduction of the Cordova whitelist system, now there is a configurable security policy to define which sites may be accessed by an application. More information about this is available in [Cordova's official Whitelist Guide](https://cordova.apache.org/docs/en/latest/guide/appdev/whitelist/).
+
+## Solution
+
+To load the iframe's content:
+
+1. Open the **iOS Config.xml** file in your project, by following the steps [outlined here](http://docs.telerik.com/platform/appbuilder/cordova/configuring-your-app/edit-configuration).
+1. In that file, add the following code before the closing `</widget>` tag: 
+ <br />`<feature name="IntentAndNavigationFilter"> `
+ <br />&ensp;&ensp;&ensp;&ensp;`<param name="ios-package" value="CDVIntentAndNavigationFilter"/>`
+ <br />&ensp;&ensp;&ensp;&ensp;`<param name="onload" value="true"/> `
+ <br />`</feature>`
+1. Finally, in the same file, you need to add the URLs the app will be navigated to:
+	1. If you want to whitelist all URLs at once, add the following line: <br />
+`<allow-navigation href="*" />`
+	1. Due to security reasons, we strongly recommend to add only the links the app will be navigated to.
+<br /> To do that, insert the desired link inside the quotes of the following line:<br />
+`<allow-navigation href="" />`<br /> You are free to have as many as needed "allow-navigation" lines.
```

Comment:
* "You are free to have as many as needed" is more blog-proper. Consider "You can add as many X as needed." or "You can add multiple X."